profile/README.md

```diff
@@ -0,0 +1,16 @@
+
+![](ogho_banner.png)
+
+# Open Source at Oracle
+
+Oracle is the purveyor of the industry’s most widely adopted open source technologies, such as Java and MySQL, and one of the founding members of the Linux Foundation, the Eclipse Foundation, and the Java Community Process.
+
+But what has guided us along the way?
+
+Our mission is to empower developers and continue supporting open source technologies. Through Oracle Cloud Infrastructure and its 37 cloud regions, we’re providing more opportunities than ever to make open source technology available to everyone.<br><br>
```

Comment:
```suggestion
Our mission is to empower developers and continue supporting open source technologies. Through Oracle Cloud Infrastructure and its global cloud regions, we’re providing more opportunities than ever to make open source technology available to everyone.
```

Having a specific number of regions means we have to keep that number up-to-date.